packages/cli-build/src/commands/build/wait.js

```diff
@@ -56,10 +57,12 @@ export class Wait extends Command {
       return this.log.info('Percy is disabled');
     }
 
-    await new PercyClient()
-      .waitForBuild(this.flags, data => {
-        this.status(data);
-      });
+    await new PercyClient({
+      clientInfo: `${pkg.name}/${pkg.version}`,
+      environmentInfo: ''
```

Comment:
@wwilsman 🚌 Strugglin' to think of what should go here. The node version is passed by the client always, so I'm not sure what would be useful here.

Also feels not great an empty string is all it takes to get around this lol

Reply:
You're right, empty environment info doesn't feel correct, but also in these instances it's already handled since client adds Node env info automatically.

I was going to suggest adding node info anyway since a `Set` is used and values can only ever be unique. However, I realized that when constructing the user-agent, node info is always added regardless if it already exists (which feels like a bug). This comment is starting to stray from this line of code, so I'll move it to the proper place. 😆